### PR TITLE
new PlayersManager with basic UI and integration with SplitScreen

### PR DESCRIPTION
**Pull request by Youenn-Bouglouan**
- new PlayersManager to create and handle Players (activate, deactivate, remove...)
- each player gets a unique Id. The id will be used to communicate with InputManager
- Simple UI with slider to choose how many players to spawn (1 to 4)
- Basic integration with SplitScreenController from Wojtek. To be reworked a bit later.
- Example of use in PipesLevelPrototype (choose number of players, spawn...)
- Using Player model from Paweł and Przemek!
- few small refactors here and there...

### CODE REVIEW

File: Assets/Scripts/Gameplay/PlayersManagement/ActivatePlayers.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            if (_splitScreenController == null)
+            {
+                DevHelper.NotificationPublisher.Publish("[ActivatePlayers] SplitScreenController not set!", Severity.Error, "Make sure the object is set within the editor.");
+            }
+
+            GameplayServices.PlayersManager.ActivateAllPlayers();
+
+            var players = GameplayServices.PlayersManager.GetAllPlayers();
+
+            // hardcoded positions for now (PipesLevelPrototype)
```
Comment (MrPeKa) [inline Assets/Scripts/Gameplay/PlayersManagement/ActivatePlayers.cs]: //TODO respawn list - prefab with list of spawn

**Comment by MrPeKa** [comment]
Maybe, add the Splitscreen and the ActivatePlayer to init scene.

File: Assets/Scripts/Gameplay/PlayersManagement/ActivatePlayers.cs (hunk 1 above)
Comment (Youenn-Bouglouan) [inline Assets/Scripts/Gameplay/PlayersManagement/ActivatePlayers.cs]: Yes, I want to do this for the next week. We will discuss where to put the spawn points with Krzysiek today.

**Comment by Youenn-Bouglouan** [comment]
Maybe, add the Splitscreen and the ActivatePlayer to init scene.
--> Splitscreen yes, however ActivatePlayers should remain on the maps in my opinion.

**Pull request merged**